.github/workflows/prerelease.yml

```diff
@@ -13,7 +13,7 @@ on:
       dependencies-to-update:
         description: 'csv of dependencies to update with the dist-tag'
         required: false
-        default: '@adobe/generator-add-action-generic'
+        default: '@adobe/generator-app-common-lib,@adobe/generator-add-web-assets-exc-react'
```

Comment:
```suggestion
        default: '@adobe/generator-app-common-lib, @adobe/generator-add-web-assets-exc-react'
```

Reply:
@roribio , not sure we need a space there, at least other templates don't have it, for example, https://github.com/adobe/generator-app-excshell/blob/3a26cfcbea52b7d11f632fbb1f340f90b6633be3/.github/workflows/prerelease.yml#L16